docs/technical-documentation.md

```diff
@@ -0,0 +1,119 @@
+<br />
+
+## Initialization
+
+The Redemptions app is initialized by passing a `Vault _vault`, `TokenManager _tokenManager`, and `address[] _redemptionTokenList`. Users are able to redeem tokens associated to the `_tokenManager` in exchange for a proportional share of each token on the `_redemptionTokenList` held in the `_vault` address.
```

Comment:
Currently, the app is initialized only with the `Vault` and `TokenManager`. The tokenList is set empty for now until we can use argument arrays from the cli

Reply:
Ok cool. Changed the docs accordingly. If we end up allowing admins to initialize the `_redemptionTokenList` via the CLI let me know